spec/features/admin_add_video_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'spec_helper'
+
+feature 'admin can add video and user can watch' do 
+  background do
+    Fabricate(:category, title: 'Science Fiction')
+  end
+  scenario 'admin can add video' do
+    sign_in_admin
+    click_link("Add a video")
+    page.should have_content "Add a New Video"
+    fill_in('video[title]', :with => "Inception")
```

Comment:
I would recommend to get rid of the old ruby hash syntax. You are using new syntax couple lines about in Fabricate method, and you should stick to one and that one should be the latest style.